### DISCUSSION TITLE
Quick edit of metabind button Templates possible?

### DISCUSSION THREAD

**Discussion by Tejeev** [mProjectsCode/obsidian-meta-bind-plugin#360] [Q&A] [answered]
Is there an easier way to edit the button Templates?
Currently my process is to:
1. Copy from the UI
2. Paste in a new file
3. Edit
4. Then go back to the Meta Bind Button Templates Editor UI and create new and paste in and save.

This is a LOT so I was looking for a file I could edit directly or something but grepping the vault dir came up with nothing.  I could not find an issue for editing these live in the UI or adding a button to edit a template in the menue where you access them, so in case one exists or there's a simple solution I'm missing, I created a discussion topic. Please point me in the right direction if this is a duplicate :)

**Comment by Tejeev** [comment]
Just saw how Painter addresses this.  Might be an option here.
See "Choose highlight colors" in the Community Plugin configuration page for Painter.
Link to Project: https://github.com/KraXen72/obsidian-painter

**Reply by mProjectsCode** [reply, marked as answer]
![image](https://github.com/mProjectsCode/obsidian-meta-bind-plugin/assets/61602848/0a5d11a9-1c31-4618-9312-145eecc81f35)

There is an edit button in the UI.

**Reply by motion2082** [reply]
You could just edit the data.json file directly with Visual Code

plugins\obsidian-meta-bind-plugin\data.json

**Reply by motion2082** [reply]
Example of data.json

![example](https://github.com/user-attachments/assets/2a4cffc7-88e7-4569-a69a-dd8b8b5bd3fa)

**Comment by Tejeev** [comment]
Ah, thank you! 
I did not know `plugins\obsidian-meta-bind-plugin\data.json` was where it was stored.  Maybe we can just change the UI interface to a direct edit of that file so I don't have to hop over to terminal just to edit my buttons?
